src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -64,6 +64,12 @@ public class ParameterProvider {
   public static final Constants.BdecParquetCompression BDEC_PARQUET_COMPRESSION_ALGORITHM_DEFAULT =
       Constants.BdecParquetCompression.GZIP;
 
+  // Iceberg mode default parameters
+  public static final int MAX_CHUNKS_IN_BLOB_AND_REGISTRATION_REQUEST_ICEBERG_MODE_DEFAULT = 1;
```

Comment:
This sounds like we're allowing to change it to something other than 1, but it has to be one for Iceberg, right? We should put this into Constants.java, everything in this file means that we're allow to override it.

Reply:
Yes. Moved to `Constants.java`.